x-pack/plugins/fleet/server/services/secrets.ts

```diff
@@ -66,7 +57,7 @@ export async function createSecrets(opts: {
   const logger = appContextService.getLogger();
   const body = values.flatMap((value) => [
     {
-      create: { _index: getSecretsIndex() },
+      create: { _index: SECRETS_INDEX },
```

Comment:
With the changes in https://github.com/elastic/kibana/pull/163075 I think you can remove the changes to this file and just focus on the test :) We switched to the new ES API instead of the using the index.